src/commands/mod.rs

```diff
@@ -1571,7 +1571,14 @@ mod tests {
         );
         assert_eq!(tx.output[0].value.to_sat(), 95_000);
         // change = 100_000 - 95_000 - /* fee without change */ 127 - /* extra fee for change output */ 43 = 4830
-        assert_eq!(warnings, vec!["Change amount of 4830 sats added to fee as it was too small to create a transaction output."]);
+        assert_eq!(
+            warnings,
+            vec![
+                "Dust UTXO. The minimal change output allowed by Liana is 5000 sats. \
+                Instead of creating a change of 4830 sats, it was added to the \
+                transaction fee. Select a larger input to avoid this from happening."
```

Comment:
Not asking to change it. But do you find this new message clearer? I'm not convinced.

Reply:
It's good that it now mentions the 5000 threshold, but otherwise it's quite similar to before I think. Using "dust UTXO" could be confusing, but like "CPFP" in the other message, it may help the user investigate further.